Packs/Yara/Playbooks/playbook-YARA_-_File_Scan_README.md

```diff
@@ -0,0 +1,44 @@
+A playbook to run YARA scan against uploaded file.
+To run the playbook, the YARA rule content and the file entry ID must be provided.
```

Comment:
Another suggestion:
To run the playbook, provide the YARA rule content and the entry ID of the file you intend to scan.